woocommerce/woocommerce-ios
# Dropped iOS 11 support

Fixes #2115 

From now we will support iOS 12 and later.
I have also solved the warnings that appeared dropping iOS 11 support.

It would be good to merge this on 4.3 release since it will enable Products M2 in beta.

## Testing
- Make sure that the `getNetworkInformation()` method in `ZendeskManager` works as before (just print `networkInformation` before and after, preferably from a real device).
- Make sure that the search controller in Shipment Providers has the same appearance as before. 

Update release notes:
